.github/workflows/build_and_test.yml

```diff
@@ -1,4 +1,4 @@
-name: Build and test [Python 3.7, 3.8, 3.9]
+name: Build and test [Python 3.7, 3.8, 3.9, 3.10]
```

Comment:
@jdebacker. Am I correct that we are keeping Python 3.7 here because Tax-Calculator is tested on Python 3.7?